isis/src/clipper/apps/jitterfit/jitterfit.xml

```diff
@@ -197,8 +250,8 @@
           Output file that holds the coefficients.
```

Comment:
Not sure if this is useful at this point, but line 239 and 226 also need descriptions (i.e. just say "SOMETHING" right now.)